README.md

```diff
@@ -31,6 +31,18 @@
   eval (fnm env --fish)
   ```
 
+### install reference
+
+```bash
+cd ~
+wget https://github.com/Schniz/fnm/releases/download/v1.1.0/fnm-linux.zip
+unzip fnm-linux.zip
+chmod +x ~/fnm-linux/fnm-linux
+ln ~/fnm-linux/fnm-linux /usr/local/bin/fnm
+echo "eval `fnm env`" >> ~/.bashrc && source ~/.bashrc
+enjoy it :)
```

Comment:
It should be clear that this is only a comment and not a command to run (a bit of a nitpick but still):

```suggestion
$ # enjoy it :)
```

Also, there should be added `$` prompts before the commands above to clarify that those are the commands users need to run in order to install `fnm`.